py-polars/tests/unit/datatypes/test_temporal.py

```diff
@@ -1031,7 +1031,6 @@ def test_supertype_timezones_4174() -> None:
     df.with_columns(df["dt_London"].shift(fill_value=date_to_fill))
 
 
-@pytest.mark.skip(reason="from_dicts cannot yet infer timezones")
```

Comment:
nearly 2 years after this skip was introduced, finally 🎉

Reply:
honestly I did this long ago enough that it might as well have been written by someone else in terms of my familiarity with it anymore.